signature/jws/signer_test.go

```diff
@@ -50,6 +46,48 @@ func TestSignWithCertChain(t *testing.T) {
 	}
 }
 
+func TestSignWithCertChain(t *testing.T) {
+	// sign with key
+	tests := []struct {
+		name string
+		fn   func() (crypto.PrivateKey, error)
+	}{
+		{
+			name: string(notation.RSA_2048),
+			fn:   func() (crypto.PrivateKey, error) { return rsa.GenerateKey(rand.Reader, 2048) },
+		},
```

Comment:
We should add some negative tests for RSA and EC with Notary unsupported key lengths where applicable, and a negative test for non RSA or EC key. Can be addressed as a separate PR.